.github/workflows/dask_test.yaml

```
@@ -0,0 +1,57 @@
+name: dask/dask tests
```

nit: 
```suggestion
name: Tests / dask/dask
```
would IMO fit better into the existing hierarchy.